server/index.js

```diff
@@ -108,17 +108,40 @@ app.post('/api/driverinfo', (req, res, next) => {
 
 app.post('/api/sendtext', (req, res, next) => {
 
-  const { toNumber, message } = req.body;
+  const sql = `
+      select "phone", "email"
+        from "users"
+        where "userId" = 1
+    `;// -replace userId with global thing
+
+  db.query(sql)
+    .then(result => {
+      const user = result.rows;
+
+      const userPhone = user[0].phone;
+      const { toNumber, message } = req.body;
+      const messages = [
+        { toNumber: toNumber, message: message },
+        { toNumber: userPhone, message: message }
+      ];
+
+      for (const item of messages) {
+        client.messages
+          .create({
+            body: item.message,
+            from: '+16193042945',
+            to: item.toNumber
+          })
+          .catch(err => {
+            next(err);
+          });
```

Comment:
for this part i recommend `Promise.all()`. That way you only use `next(err)` one time if either fails

```js
Promise.all([
  client.messages.create({ ... }),
  client.messages.create({ ... })
])
.then(() => res.json({ success: true }))
.catch(err => next(err))
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all